documentation/docs/16-assets.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Asset handling
+---
+
+### Hashing
+
+To include hashes in your asset file names and cache them, you can have Vite process your assets by importing them as shown below:
+
+```html
+<script>
+	import imageSrc from '$lib/assets/image.png';
+</script>
+
+<img src="{imageSrc}" />
+```
+
+If you prefer to directly import in the markup, you can use a preprocessors to assist with this such as [svelte-preprocess-import-assets](https://github.com/bluwy/svelte-preprocess-import-assets) or [svelte-image](https://github.com/matyunya/svelte-image).
+
+For assets included via `url()` in you may find the [`experimental.useVitePreprocess`](https://github.com/sveltejs/vite-plugin-svelte/blob/main/docs/config.md#usevitepreprocess) option in `vite-plugin-svelte` to be useful.
+
+### Optimization
+
+You may wish to utilize compressed image formats such as `.webp` or `.avif` or responsive images that serve a different size based on your device's screen. For images that are included statically in your project you may use a preprocessor like [svelte-image](https://github.com/matyunya/svelte-image) or Vite plugin such as [vite-imagetools](https://github.com/JonasKruckenberg/imagetools).
```

Comment:
I realise it's out of scope of this PR but it'd be great if we had a canonical recommendation, rather than 'you could try this thing or that thing 🤷'

```suggestion
You may wish to utilize compressed image formats such as `.webp` or `.avif` or responsive images that serve a different size based on your device's screen. For images that are included statically in your project you may use a preprocessor like [svelte-image](https://github.com/matyunya/svelte-image) or a Vite plugin such as [vite-imagetools](https://github.com/JonasKruckenberg/imagetools).
```

Reply:
totally agree